packages/main/src/TimePicker.hbs

```diff
@@ -0,0 +1,73 @@
+<div id="{{this._id}}" formatPattern="{{formatPattern}}" class="ui5-timepicker">
+	<ui5-input
+			id="{{_id}}-inner"
+			value="{{value}}"
+			?disabled="{{disabled}}"
+			?readonly="{{readonly}}"
+			value-state="{{valueState}}"
+			@ui5-change="{{_handleInputChange}}"
+			@ui5-input="{{_handleInputLiveChange}}"
+	>
+		<ui5-icon
+			slot="icon"
+			name="time-entry-request"
+			tabindex="-1"
+			show-tooltip
+			@click="{{togglePicker}}"
+			input-icon
+			?pressed="{{_isPickerOpen}}"
+		></ui5-icon>
```

Comment:
When the popover is opened, the icon should have toggled background (dark blue in fiori 3), but it does not

Reply:
Done